pkg/kapp/diffgraph/change_group_name.go

```diff
@@ -0,0 +1,78 @@
+// Copyright 2020 VMware, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+package diffgraph
+
+import (
+	"fmt"
+	"regexp"
+	"strings"
+
+	ctlres "github.com/k14s/kapp/pkg/kapp/resources"
+	ctlcrd "github.com/k14s/kapp/pkg/kapp/resourcesmisc"
+)
+
+type ChangeGroupName struct {
```

Comment:
@cppforlife Just realised that we are also using the `AsString()` method to[ parse change rules](https://github.com/vmware-tanzu/carvel-kapp/pull/310/commits/2fc07f72b807def6bacc5d57f264b7cb7698ac8e#diff-72f6f14971e177e069bd13bca3d398b63f3896678b931fc1e57f873dedec2762R167) (the change group names contained in them) 
Should we consider a better name maybe?